.github/workflows/shippable_builds.yml

```diff
@@ -382,7 +383,21 @@ jobs:
 
           # Add changelogs, build version changes and metadata symlink
           git add ./app-metadata/${APPLICATION_ID}/en-US/changelogs/*
-          git add ./app-${APP_NAME}/src/main/res/raw/changelog_master.xml
+          if [[ "$APP_NAME" = "k9mail" ]]; then
+            git add ./app-${APP_NAME}/src/main/res/raw/changelog_master.xml
+          elif [[ "$APP_NAME" = "thunderbird" ]]; then
+            case "$RELEASE_TYPE" in
+              "daily")
+                git add ./app-${APP_NAME}/src/daily/res/raw/changelog_master.xml
+                ;;
+              "beta")
+                git add ./app-${APP_NAME}/src/beta/res/raw/changelog_master.xml
+                ;;
+              "release")
+                git add ./app-${APP_NAME}/src/release/res/raw/changelog_master.xml
+                ;;
```

Comment:
Any reason not to use $RELEASE_TYPE in the path name here? 

Good to go either way though.

Reply:
That would have been a lot simpler.